src/main/java/net/mcreator/ui/dialogs/tools/WoodPackMakerTool.java

```diff
@@ -0,0 +1,432 @@
+		fenceGateBlock.texture = planksTextureName;
+		fenceGateBlock.soundOnStep = new StepSound(workspace, "WOOD");
+		fenceGateBlock.hardness = 2 * factor;
+		fenceGateBlock.resistance = 3 * factor;
+		fenceGateBlock.breakHarvestLevel = 0;
+		fenceGateBlock.flammability = (int) Math.round(5 * factor);
+		mcreator.getWorkspace().getModElementManager().storeModElementPicture(fenceGateBlock);
+		mcreator.getWorkspace().addModElement(fenceGateBlock.getModElement());
+		mcreator.getWorkspace().getGenerator().generateElement(fenceGateBlock);
+		mcreator.getWorkspace().getModElementManager().storeModElement(fenceGateBlock);
+
+		//Tag - Items
+		//Mainly used for recipes and loot tables
+		Tag woodItemTag = (Tag) ModElementTypeRegistry.REGISTRY.get(ModElementType.TAG)
+				.getModElement(mcreator, new ModElement(workspace, name + "ItemsTag", ModElementType.TAG), false)
+				.getElementFromGUI();
+		woodItemTag.namespace = "forge";
+		woodItemTag.name = RegistryNameFixer.fix(name) + "_log";
+		woodItemTag.type = "Items";
+		woodItemTag.items = new ArrayList<>();
+		woodItemTag.items.add(new MItemBlock(workspace, "CUSTOM:" + woodBlock.getModElement().getName()));
+		woodItemTag.items.add(new MItemBlock(workspace, "CUSTOM:" + logBlock.getModElement().getName()));
+		mcreator.getWorkspace().getModElementManager().storeModElementPicture(woodItemTag);
+		mcreator.getWorkspace().addModElement(woodItemTag.getModElement());
+		mcreator.getWorkspace().getGenerator().generateElement(woodItemTag);
+		mcreator.getWorkspace().getModElementManager().storeModElement(woodItemTag);
+
+		//Recipes
+		Recipe woodRecipe = (Recipe) ModElementTypeRegistry.REGISTRY.get(ModElementType.RECIPE)
+				.getModElement(mcreator, new ModElement(workspace, name + "WoodRecipe", ModElementType.RECIPE),
+						false).getElementFromGUI();
+		woodRecipe.recipeSlots[0] = new MItemBlock(workspace, "CUSTOM:" + logBlock.getModElement().getName());
+		woodRecipe.recipeSlots[1] = new MItemBlock(workspace, "CUSTOM:" + logBlock.getModElement().getName());
+		woodRecipe.recipeSlots[3] = new MItemBlock(workspace, "CUSTOM:" + logBlock.getModElement().getName());
+		woodRecipe.recipeSlots[4] = new MItemBlock(workspace, "CUSTOM:" + logBlock.getModElement().getName());
+		woodRecipe.recipeReturnStack = new MItemBlock(workspace, "CUSTOM:" + name + "Wood");
+		woodRecipe.recipeRetstackSize = 3;
+		mcreator.getWorkspace().getModElementManager().storeModElementPicture(woodRecipe);
+		mcreator.getWorkspace().addModElement(woodRecipe.getModElement());
+		mcreator.getWorkspace().getGenerator().generateElement(woodRecipe);
+		mcreator.getWorkspace().getModElementManager().storeModElement(woodRecipe);
+
+		Recipe planksRecipe = (Recipe) ModElementTypeRegistry.REGISTRY.get(ModElementType.RECIPE)
+				.getModElement(mcreator, new ModElement(workspace, name + "PlanksRecipe", ModElementType.RECIPE),
+						false).getElementFromGUI();
+		planksRecipe.recipeSlots[4] = new MItemBlock(workspace, "TAG:" + woodItemTag.name);

```

Comment:
You have to add the tag's namespace (which in this case is "forge") to the tag (like: "TAG:forge:cherry_log")